src/main/java/strategy/handlers/ReturnHandler.java

```diff
@@ -0,0 +1,12 @@
+package strategy.handlers;
+
+import db.Storage;
+import model.FruitTransaction;
+import strategy.OperationHandler;
+
+public class ReturnHandler implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction transaction) {
+        Storage.fruitStorage.merge(transaction.getFruit(), transaction.getQuantity(), Integer::sum);

```

Comment:
Direct access to `Storage.fruitStorage` breaks encapsulation. It's better to interact with storage through a service layer or use methods that encapsulate the underlying data structure.